internal/statestore/backfill_test.go

```diff
@@ -604,3 +604,57 @@ func generateBackfills(ctx context.Context, t *testing.T, service Service, amoun
 
 	return backfills
 }
+
+func TestCleanupBackfills(t *testing.T) {
+	cfg, closer := createRedis(t, false, "")
+	defer closer()
+	service := New(cfg)
+	require.NotNil(t, service)
+	defer service.Close()
+	ctx := utilTesting.NewContext(t)
+
+	rc, err := redis.Dial("tcp", fmt.Sprintf("%s:%s", cfg.GetString("redis.hostname"), cfg.GetString("redis.port")))
+	require.NoError(t, err)
+
+	bfID := "mockBackfill-1"
+	ticket1ID := "t1"
+	ticket2ID := "t2"
+	ticketIDs := []string{ticket1ID, ticket2ID}
```

Comment:
nit - since ticket1ID, ticket2ID are not used anywhere else, you might as well do []string{"t1", "t2"}

Reply:
ok